### PR TITLE
add tcpdump example + exfiltration via email

### PR DESCRIPTION
**Pull request by julesbozouklian**
add simple tcpdump example with exfiltration via email

### CODE REVIEW

File: payloads/library/recon/simple-tcpdump/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+- Install following packages : ``` coreutils-timeout zip ```
+- Refer to this payload to install package https://github.com/julesbozouklian/shark_jack_payload/blob/main/payload/util/install_package.sh
```
Comment (foxtrot) [inline payloads/library/recon/simple-tcpdump/README.md:2]: Any chance you could put this into this PR instead? Would be easier to update in the future.

File: payloads/library/recon/simple-tcpdump/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+- Or SSH to the Shark jack and use following command : ``` opkg install coreutils-timeout zip ```
+
+
+- For the mail refer to this exemple :
+https://github.com/julesbozouklian/shark_jack_payload/tree/main/payload/exfiltrate/email
```
Comment (foxtrot) [inline payloads/library/recon/simple-tcpdump/README.md:7]: See above

File: payloads/library/recon/simple-tcpdump/payload.sh [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+# LED FAIL (Red Blink)... Failed to update opkg or install package
+# LED FINISH (Green Fast Blink to Solid)... Package install or listsuccessful
+#
+
+LOG_DIR=/root/loot/recon/simple-tcpdump
+DATE=`date +"%Y-%m-%d"`
+TIMESTAMP=`date +"%Y-%m-%d %T"`
+INTERFACE="eth0"
+TIMEOUT="15"
+EMAIL_RECEIPT="jules.bozouklian@epitech.eu"
```
Comment (foxtrot) [inline payloads/library/recon/simple-tcpdump/payload.sh:18]: Are you sure you don't want to change this? :)

**Review by foxtrot — CHANGES_REQUESTED** [review]

File: payloads/library/recon/simple-tcpdump/README.md (hunk 1 above)
Comment (julesbozouklian) [reply payloads/library/recon/simple-tcpdump/README.md:2]: yes i update my PR

Comment (julesbozouklian) [reply payloads/library/recon/simple-tcpdump/README.md:2]: ok it's updated

**Comment by foxtrot** [comment]
Thanks!

**Pull request merged**